python/ray/data/preprocessors/torch.py

```diff
@@ -120,3 +121,6 @@ def transform_batch(batch: np.ndarray) -> np.ndarray:
             outputs = transform_batch(np_data)
 
         return outputs
+
+    def preferred_batch_format(cls) -> BatchFormat:
+        return BatchFormat.NUMPY
```

Comment:
`TorchVisionPreprocessor` doesn't implement `_transform_pandas`. Do we need to override `preferred_batch_format`? (Not saying it's an issue, just checking my understanding)

Reply:
Yep you're right, it's not strictly necessary!